packages/cli/src/__tests__/artificial-panic.test.ts

```diff
@@ -1,13 +1,17 @@
 import { jestContext } from '@prisma/get-platform'
 import { serialize } from '@prisma/get-platform/src/test-utils/jestSnapshotSerializer'
-import { getDMMF, isRustPanic } from '@prisma/internals'
+import { getDMMF, isRustPanic, MultipleSchemas, relativizePathInPSLError } from '@prisma/internals'
 import { DbPull } from '@prisma/migrate'
 
 import { Format } from '../Format'
 import { Validate } from '../Validate'
 
 const ctx = jestContext.new().assemble()
 
+function sanitizeSchemas(schemas: MultipleSchemas | undefined): MultipleSchemas | undefined {
+  return schemas?.map(([path, content]) => [relativizePathInPSLError(path), content])
+}
```

Comment:
Maybe add a comment about why we need to relative paths in PSL?
Iirc, it's to ensure the paths displayed in Prisma snapshot tests remain in "relative" form, because `prisma-engines` would treat them as absolute paths instead.